sky/engine/core/painting/RRect.dart

```diff
@@ -0,0 +1,32 @@
+part of dart_ui;
+
+// An immutable rounded rectangle.
+class RRect {
+  RRect();
+
+  final Float32List _value = new Float32List(6);
+
+  /// Initialize with the same radii for all four corners.
+  void setRectXY(Rect rect, double xRad, double yRad) {
```

Comment:
IMHO, we should make RRect immutable, like Rect, Size, and Point.  Maybe we should prepare to make that change in this patch by adding a constructor:

```dart
RRect.fromRectXY(Rect rect, xRadius, yRadius)
```

After this patch lands, we can convert callers over to that and then remove the setRectXY...